modules/swagger-parser-v3/src/main/java/io/swagger/parser/v3/util/OpenAPIDeserializer.java

```diff
@@ -1,23 +1,2081 @@
+        info.setLicense(license);
+
+        value = getString("version", node, true, location, result);
+        info.setVersion(value);
+
+        info.setExtensions(new LinkedHashMap<>());
+
+        Set<String> keys = getKeys(node);
+        for(String key : keys) {
+            if(key.startsWith("x-")) {
+                info.getExtensions().put(key, Json.mapper().convertValue(node.get(key), Object.class));
+            } else if (!INFO_KEYS.contains(key)) {
+                result.extra(location, key, node.get(key));
+            }
+        }
+
+        return info;
+    }
+
+    public License getLicense(ObjectNode node, String location, ParseResult result) {
+        if (node == null)
+            return null;
+
+        License license = new License();
+
+        String value = getString("name", node, true, location, result);
+        license.setName(value);
+
+        value = getString("url", node, false, location, result);
+        license.setUrl(value);
+
+        license.setExtensions(new LinkedHashMap<>());
+
+        Set<String> keys = getKeys(node);
+        for(String key : keys) {
+            if(key.startsWith("x-")) {
+                license.getExtensions().put(key, Json.mapper().convertValue(node.get(key), Object.class));
+            }  else if (!LICENSE_KEYS.contains(key)) {
+                result.extra(location + ".license", key, node.get(key));
+            }
+        }
+
+        return license;
+    }
+
+    public Contact getContact(ObjectNode node, String location, ParseResult result) {
+        if (node == null)
+            return null;
+
+        Contact contact = new Contact();
+
+        String value = getString("name", node, false, location + ".name", result);
+        contact.setName(value);
+
+        value = getString("url", node, false, location + ".url", result);
+        contact.setUrl(value);
+
+        value = getString("email", node, false, location + ".email", result);
+        contact.setEmail(value);
+
+        contact.setExtensions(new LinkedHashMap<>());
+
+        Set<String> keys = getKeys(node);
+        for(String key : keys) {
+            if(key.startsWith("x-")) {
+                contact.getExtensions().put(key, Json.mapper().convertValue(node.get(key), Object.class));
+            } else if (!CONTACT_KEYS.contains(key)) {
+                result.extra(location + ".contact", key, node.get(key));
+            }
+        }
+
+        return contact;
+    }
+
+    public Content getContent(ObjectNode node, String location, ParseResult result){
+        if (node == null) {
+            return null;
+        }
+        Content content = new Content();
+        Set<String> keys = getKeys(node);
+        for(String key : keys) {
+            MediaType mediaType = getMediaType((ObjectNode) node.get(key), location, result);
+            if (mediaType != null) {
+                content.addMediaType(key, mediaType);
+            }
+        }
+
+        return content;
```

Comment:
Check to see if any values are required, and if not present set an error in the `ParseResult`